File: core-commons/src/main/java/net/blueshell/common/dto/TelemetryDTO.java
package net.blueshell.common.dto;


import lombok.Data;
import lombok.EqualsAndHashCode;
import net.blueshell.common.Image;
import net.blueshell.common.enums.PlatformType;

import java.sql.Timestamp;
import java.util.List;

@Data
@EqualsAndHashCode(callSuper = true)
public class TelemetryDTO extends BaseDTO {
    private String id;
    private String url;
    private PlatformType platform;
    private Timestamp createdAt;
}

File: api/src/main/kotlin/net/blueshell/api/telemetry/web/dto/TelemetryDTO.kt
package net.blueshell.api.telemetry.web.dto

import io.swagger.v3.oas.annotations.media.Schema
import net.blueshell.api.shared.enums.PlatformType
import net.blueshell.api.shared.dto.AuditedAutoIdDTO

@Schema(name = "Telemetry")
data class TelemetryDTO(
    var url: String? = null,
    var platform: PlatformType? = null
) : AuditedAutoIdDTO()
